.github/CHANGELOG.md

```diff
@@ -45,22 +44,31 @@
   Mach-Zehnder interferometers corresponding to on-chip phases.
   [(#363)](https://github.com/XanaduAI/strawberryfields/pull/363)
 
-* Changes the `number_expectation` method for the `BaseFockState` class to be an instance
-  of `diagonal_expectation`.
+* Changes the `number_expectation` method for the `BaseFockState` class to be an
+  instance of `diagonal_expectation`.
   [(#389)](https://github.com/XanaduAI/strawberryfields/pull/389)
 
+* Increases the speed at which the following gates are generated: `Dgate`, `Sgate`,
+  `BSgate` and `S2gate` by relying on a new recursive implementation.
+  This has substantial effects on the speed of the `Fockbackend` and the `TFbackend`,
```

Comment:
I think the usage of the recursive gates in the `Fock` backend was already there in a different PR.

Reply:
hm. good point